pkg/operator/ceph/cluster/controller.go

```diff
@@ -505,6 +613,7 @@ func (c *ClusterController) onUpdate(oldObj, newObj interface{}) {
 	// Display success after upgrade
 	if versionChanged {
 		printOverallCephVersion(c.context, cluster.Namespace)
+		// TODO: Update all the crd controllers that there is a new version of the ceph image to deploy
```

Comment:
The loop that calls `child.ParentClusterChanged()` will already notify the child controllers with every orchestration. I don't think this is needed here.